validator/main.py

```diff
@@ -173,14 +183,31 @@ def validate(
             not self._disable_classifier and not self._disable_llm
         ):  # Use ensemble (Zero-Shot + Ensemble)
             applicable_topics = self.get_topics_ensemble(value, list(invalid_topics))
+            if not applicable_topics:
```

Comment:
This PassResult generation should apply to all configurations of `disable_llm` and `disable_classifier`, and not just the ensemble method.